ui/tests/pages/jobs/list.js

```diff
@@ -64,6 +64,17 @@ export default create({
     }),
   },
 
+  pageSizeSelect: {
+    isPresent: isPresent('[data-test-page-size-select]'),
+    open: clickable('[data-test-page-size-select] .ember-power-select-trigger'),
+    selectedOption: text('[data-test-page-size-select] .ember-power-select-selected-item'),
+    options: collection('.ember-power-select-option', {
+      testContainer: '#ember-testing',
+      resetScope: true,
+      label: text(),
+    }),
+  },
+
```

Comment:
This looks like the third repetition of the identical page object component, maybe worthy of extraction?

Reply:
I agree.

Reply:
Officially extracted.